src/lib/components/Swap/Toolbar.tsx

```diff
@@ -118,6 +121,14 @@ export default function Toolbar({ disabled }: { disabled?: boolean }) {
           </>
         )
       }
+      if (routeNotFound && inputCurrency && outputCurency && !routeIsLoading && !routeIsSyncing) {
```

Comment:
`routeNotFound` is a liiittle confusing here, since we're doing boolean checks for other negative statements w/ !

Reply:
ill change to isRouteFound and reverse